releasenotes/notes/fix-openai-patch-process-response-b52430a78751de24.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    openai: This fix resolves an issue where an internal OpenAI method was not being patched correctly and
```

Comment:
Can we call out the method that we failed to patch. This might help users correlate the error message to the fix